packages/forms/src/UIForm/fields/Code/Code.component.js

```diff
@@ -172,8 +172,20 @@ try {
 
 	CodeWidget = Code;
 } catch (error) {
-	// eslint-disable-next-line no-console
-	console.warn('CodeWidget react-ace not found, fallback to Textarea');
+	class WrappedTextArea extends React.PureComponent {
+		// eslint-disable-line react/no-multi-comp
```

Comment:
This one should be up to line 175 I guess with 
`// eslint-disable-next-line react/no-multi-comp`

Reply:
Good catch, IDE got me confused